RELEASE_NOTES.md

```diff
@@ -91,7 +91,32 @@ bin/rails decidim:upgrade:user_groups:remove_groups_notifications
 
 You can read more about this change on PR [#14130](https://github.com/decidim/decidim/pull/14130).
 
-### 2.3. [[TITLE OF THE ACTION]]
+### 2.3. Automatic deletion of inactive participant accounts
```

Comment:
This is ok, but this cron doesn't work properly, as there is no information about the sidekiq queue.